tower-grpc-interop/Dockerfile-server-go

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.9
+
+ENV PACKAGE google.golang.org/grpc/interop/server
+ENV PACKAGEDIR $GOPATH/src/$PACKAGE
+
+WORKDIR /
```

Comment:
what is needed from the workdir? As it is, this line bsaically ensures that you don't cache the rest of the layers below this

Reply:
this is just a case of me not knowing a whole lot about Docker; @siggy sent me an example dockerfile for a Go program and this is based directly on that. will fix.

Reply:
Addressed in 933a6875d4fc700ac25e35e834291915c221a5cf.